lib/edoc/src/edoc_extract.erl

```diff
@@ -230,6 +230,14 @@ add_macro_defs(Defs0, Opts, Env) ->
 
 %% INHERIT-OPTIONS: text/4
 
+-spec file(File, Context, Env, Options) -> {ok, Tags} | {error, Reason} when
+      File :: filename(),
+      Context :: overview,
```

Comment:
Since there is no way to perfectly know in which way these two functions are used from outside of otp and for the sake of documentation and typing, I lean towards listing the 5 atoms. At leat it's better than `atom()`